crates/host/src/wasmbus/mod.rs

```diff
@@ -231,6 +266,14 @@ async fn resolve_target(
             .and_then(|targets| targets.get(DEFAULT_LINK_NAME))
             .context("link not found")?
             .clone(),
+        Some(TargetEntity::Wrpc(target)) => {
+            let (namespace, package, _) = target.interface.as_parts();
+            WasmCloudEntity {
+                public_key: target.id.clone(),
+                contract_id: format!("{namespace}:{package}"),
+                link_name: target.link_name.clone(),
```

Comment:
Are we leaving `WasmCloudEntity` & `TargetEntity` in? Ceratinly avoids excessive breakage so if this is how we're going to do it :+1: , I'll revert my changes that remove these.